edgepi-relay.js

```diff
@@ -0,0 +1,50 @@
+module.exports = function (RED) {
+    const rpc = require('@edgepi-cloud/edgepi-rpc')
+  
+    function RelayNode(config) {
+      // Create new node instance with user config
+      RED.nodes.createNode(this, config);
+      const node = this;
+      const ipc_transport = "ipc:///tmp/edgepi.pipe"
+      const tcp_transport = `tcp://${config.tcpAddress}:${config.tcpPort}`
+      const transport = (config.transport === "Network") ? tcp_transport : ipc_transport;
```

Comment:
instead of assuming that any transport that is not "Network" is automatically IPC, i think it's better to be explicit:

```
if (config.transport === "Network") {
    transport = tcp_transport
} else if (config.transport === "<your-IPC-network-name>") {
    transport = ipc_transport
} else {
    // throw some kind of error
}
```

this way we future proof it in case we add support for a new transport in the future

Reply:
mmm, right now it is hard set in the UI to only be configurable to `local` (which is hard-coded ipc, and can't be changed), and network  which is tcp (where they can enter the address). In the future, network will likely have multiple sub options other than tcp but for now I think this is okay.

Reply:
Basically if `local` is selected for `Rpc Server`, then no address field inputs are shown and ipc is used. If `Network` is selected, then users can enter address for tcp transport

Reply:
i see, that makes sense